code/drasil-lang/Language/Drasil/Chunk/UnitDefn.hs

```diff
@@ -56,6 +56,9 @@ instance IsUnit        UnitDefn where
 class MayHaveUnit u where
    getUnit :: u -> Maybe UnitDefn
 
+class HasUnit u where
```

Comment:
Can you change the name of this class? It clashes with another of the same name. It should be temporary too.  So `TempHasUnit`?